evaporate.js

```diff
@@ -597,7 +597,6 @@
                         setStatus(COMPLETE);
                     } else {
                         l.d('headObject not found on S3.');
-                        me.name = awsKey;
```

Comment:
Sam, can you help me understand why this was problematic. You may be correct but I want to understand if this line was indeed an oversight and not something required. I think in the end, if this line needs to be removed, it's not due to IE but because it shouldn't be there at all.